shell/packages/sandstorm-db/db.js

```diff
@@ -153,6 +153,7 @@ Packages = new Mongo.Collection("packages");
 //   url:  When status is "download", the URL from which the SPK can be obtained, if provided.
 //   isAutoUpdated: This package was downloaded as part of an auto-update. We shouldn't clean it up
 //     even if it has no users.
+//   isPreinstalled: This package was downloaded in order to be pre-installed by users.
```

Comment:
Note that the dynamic post-install check would be: "If app ID is marked for pre-install *and* the package ID is the most recent one listed in the app index"